packages/lib/CalEventParser.ts

```diff
@@ -174,7 +174,7 @@ export const getRescheduleLink = (calEvent: CalendarEvent): string => {
   const Uid = getUid(calEvent);
   const seatUid = getSeatReferenceId(calEvent);
 
-  return `${WEBAPP_URL}/reschedule/${seatUid ? seatUid : Uid}`;
+  return `${calEvent.bookerUrl ?? WEBAPP_URL}/reschedule/${seatUid ? seatUid : Uid}`;
```

Comment:
For Org
<img width="1547" alt="image" src="https://github.com/calcom/cal.com/assets/1780212/2bf812f3-1b84-4625-9873-cbde4a3ab020">

For non org
<img width="1328" alt="image" src="https://github.com/calcom/cal.com/assets/1780212/c3660960-5bba-4db2-b313-81bef2287d5e">